### PR TITLE
Remove links to deprecated OV notebooks

### PR DESCRIPTION
**Pull request by nikita-savelyevv**
### Reason for changes

Some notebooks were removed with https://github.com/openvinotoolkit/openvino_notebooks/pull/1987

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -280,10 +280,8 @@ A collection of ready-to-run Jupyter* notebooks tutorials and demos are availabl
...
 | [PyTorch Training-Time Compression](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/pytorch-quantization-aware-training)                                                                                                                                                                                                   |                                Training-Time Compression                                |  PyTorch   |        Image Classification         |
-| [TensorFlow Training-Time Compression](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/tensorflow-training-openvino)                                                                                                                                                                                                       |                                Training-Time Compression                                | Tensorflow |        Image Classification         |
```
Comment (alexsu52) [inline README.md:286]: https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/tensorflow-quantization-aware-training

File: README.md [hunk 2]
```diff
@@ -280,10 +280,8 @@ A collection of ready-to-run Jupyter* notebooks tutorials and demos are availabl
...
 | [PyTorch Model Quantization](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/pytorch-post-training-quantization-nncf)                                                                                                                                                                                                      |                               Post-Training Quantization                                |  PyTorch   |        Image Classification         |
-| [TensorFlow Model Quantization](https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/tensorflow-training-openvino)                                                                                                                                                                                                              |                               Post-Training Quantization                                | Tensorflow |        Image Classification         |
```
Comment (alexsu52) [inline README.md:283]: Please, file a ticket to create separate PTQ notebook for Tensorflow.

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (nikita-savelyevv) [reply README.md:283]: Created 140332

**Review by alexsu52 — APPROVED** [review]
LGTM

**Review by MaximProshin — APPROVED** [review]

**Pull request merged**